src/core/seat/hotspot-manager.hpp

```diff
@@ -49,6 +49,11 @@ class hotspot_instance_t : public noncopyable_t
     /** Timer for hotspot activation */
     wf::wl_timer timer;
 
+    /** Only one event should be triggered once the cursor enters the hotspot area.
```

Comment:
Small nitpick: if comment is not single-line, text should begin on a new line

Reply:
Like this?

```
    /**
     *  Only one event should be triggered once the cursor enters the hotspot area.
     *  This prevents another event being fired until the cursor has left the area.
     */
```

Reply:
Yes, but then you'd need one less identation space too :)

Reply:
Fixed :)